app/models/transportation_office.rb

```diff
@@ -1,9 +1,9 @@
-class TransportationOffice < Office
-  # rubocop:disable Rails/InverseOf
-  belongs_to :shipping_office, foreign_key: :shipping_office_id, optional: true
-  # rubocop:enable Rails/InverseOf
+class TransportationOffice < ApplicationRecord
+  include Locatable
 
-  def self.by_distance_with_shipping_office(latitude:, longitude:)
-    includes(:shipping_office).by_distance(origin: [latitude, longitude])
-  end
+  belongs_to :shipping_office, optional: true
+
+  validates :name, presence: true
+
+  default_scope { includes(:shipping_office) }
```

Comment:
Seems reasonable to have this here for now since we're _always_ showing a Shipping Office (where one exists) when displaying Transportation Offices.